src/assets/icons/resource-type-root-fill.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512">
+   <!-- MIT License: https://opensource.org/licenses/MIT -->
```

Comment:
sorry, I meant adding another comment here in the svg file, linking yours and @elizavetaRa github profiles as authors. just something like:
`<!-- author: https://github.com/elizavetaRa -->`

Reply:
Ah, that makes sense! I've just added that.